doc/content/gateways/COTX-X1 Lite gateway

```diff
@@ -0,0 +1,47 @@
+---
+title: "COTX X1 Lite"
+description: ""
+---
+
+COTX X1 Lite is a standard LoRaWAN indoor gateway and a Helium data-only hotspot.
+It is highly compatible and has a user-friendly interface.
+Rich in software tools, powerful back office management and technical support.
+Compact and exquisite appearance, easy to install and deploy.
+![Lite-1.png](./TTN-X1%20Lite-1.png)
+
+##Key Features 
```

Comment:
```suggestion
## Key Features 
```

Reply:
If there is no this space, the docs don't render properly. Please apply this fix for all occurrences.